checklists/lz_checklist.en.json

```diff
@@ -1511,11 +1511,12 @@
         {
             "category": "Security, Governance and Compliance",
             "subcategory": "Governance",
-            "text": "Assign the built-in Policy Contributor role at a particular scope to enable application-level governance.",
+            "text": "Assign the built-in Resource Policy Contributor role at a particular scope to enable application-level governance.",
+            "description": "Assigning the Resource Policy Contributor role to specific scopes allows you to delegate policy management to relevant teams. For instance, a central IT team may oversee subscription-level policies, while application teams handle policies for their resource groups, enabling distributed governance with adherence to organizational standards.",
```

Comment:
I wonder if this should be changed to have central IT having access to the Management Group level, and application teams for Subscriptions, because that would be more in line with the principle of subscription delegation?

I can see it both ways, but I wanted to raise it for discussion.